src/components/MaterialUI/DataDisplay/collapsableListProps.js

```diff
@@ -1,29 +1,31 @@
 import ComponentProps from "../componentProps";
 
 class CollapsableListProps extends ComponentProps {
-  static propNames = {
-    isExpanded: "isExpanded",
-    hasMessage: "hasMessage",
-    openMessage: "openMessage",
-    closeMessage: "closeMessage",
-    expandPosition: "expandPosition",
-  };
+	static propNames = {
+		isExpanded: "isExpanded",
+		hasMessage: "hasMessage",
+		openMessage: "openMessage",
+		closeMessage: "closeMessage",
+		expandPosition: "expandPosition",
+		maxContent: "maxContent",
+	};
 
-  constructor() {
-    super();
-    this.componentProps.set(this.constructor.propNames.isExpanded, null);
-    this.componentProps.set(this.constructor.propNames.hasMessage, null);
-    this.componentProps.set(this.constructor.propNames.openMessage, null);
-    this.componentProps.set(this.constructor.propNames.closeMessage, null);
-    this.componentProps.set(this.constructor.propNames.expandPosition, null);
+	constructor() {
+		super();
+		this.componentProps.set(this.constructor.propNames.isExpanded, null);
+		this.componentProps.set(this.constructor.propNames.hasMessage, null);
+		this.componentProps.set(this.constructor.propNames.openMessage, null);
+		this.componentProps.set(this.constructor.propNames.closeMessage, null);
+		this.componentProps.set(this.constructor.propNames.expandPosition, null);
+		this.componentProps.set(this.constructor.propNames.maxContent, true);
```

Comment:
default values should be assigned inside component. here should be null as for others